src/index.css

```diff
@@ -0,0 +1,187 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(./images/cyptobg.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+body::before {
+  display: block;
+  content: '';
+  height: 110px;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+a {
+  text-decoration: none;
+}
+
+.autoimg {
+  padding: 9px 0 9px 0;
+}
+
+.header {
+  display: flex;
+  position: fixed;
+  top: 0;
+  background-color: #002855;
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+}
```

Comment:
- [ ] This is meant to be a mobile application and one of the requirements is that your design should follow the wireframe provided. The navbar of both the Home and Details page on mobile view isn't identical to that of the template given. Kindly adjust your design.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/211052668-dcf0c391-c100-4d32-939d-a8be8978658b.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/211053262-99933335-517a-4830-adf2-d7969ec13282.png)

#### Hint: Notice the placement of the icons. They are aligned in a row. 👍